<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Lenovo-PC" ignored="0" name="Default test(failed)" tests="2" failures="2" timestamp="31 Mar 2020 15:13:30 GMT" time="96.729" errors="0">
  <testcase name="login" time="31.668" classname="com.ivavsys.TestPOM.LoginTest">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at com.ivavsys.pom.EnterTimeTrack.Vaildate(EnterTimeTrack.java:23)
at com.ivavsys.TestPOM.LoginTest.login(LoginTest.java:32)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- login -->
  <testcase name="login" time="32.231" classname="com.ivavsys.TestPOM.LoginTest">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at com.ivavsys.pom.EnterTimeTrack.Vaildate(EnterTimeTrack.java:23)
at com.ivavsys.TestPOM.LoginTest.login(LoginTest.java:32)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- login -->
</testsuite> <!-- Default test(failed) -->
